docs/alerts/monitors/create-monitor.md

```diff
@@ -262,20 +262,20 @@ Alert Name allows you to customize the name that appears on the Alert Page. By d
 ### Evaluation Delay
 Collection delays may occur due to your environment and it takes a couple of minutes for data to be processed into Sumo Logic. Since Monitors run on data from the most current time period, it's possible for Monitors to evaluate against incomplete data. As a result, Monitors can generate false positives or negatives that can cause confusion. Set an evaluation delay in seconds to delay the evaluation of a Monitor, so it doesn't look at the most current time (where data can be incomplete) and instead looks at an older period of time, where you have more complete data.<br/> ![additional settings evaluation delay.png](/img/monitors/additional-settings-evaluation-delay.png)<br/>If your data is coming from the [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md) we recommend a setting of 900 seconds.
 
-## Step 3: Notifications
-When a trigger condition is met you can send notifications to other people and services. To add notifications, click on the **Add Notification** button. You can add more than one notification channel for a Monitor.<br/>  ![monitor notifications input.png](/img/monitors/monitor-notifications-input.png)
+## Step 3: Notifications (optional)
 
-Metric Monitors have an option to send notifications either as a group or separately. **Group Notifications** define whether you want single notifications per time series that match the Monitor query or you want group notifications where you receive a single notification for the entire Monitor. Log Monitors always group notifications.
+When a trigger condition is met, you can send notifications to other people and services. Metrics monitors have an option to send notifications either as a group or separately. **Group Notifications** define whether you want single notifications per time series that match the Monitor query or you want group notifications where you receive a single notification for the entire Monitor. Log monitors always group notifications.
```

Comment:
This is old way. With alert grouping, this text is not correct anymore. Can you suggest @rmamilla-sumo

Reply:
Yes there is an additional option for configuring grouping fields and alerts trigger independently within each grouping field.

Reply:
I'm going to create a separate ticket for this as it's unrelated to the timezone release, so we can get this out the door. And we'll edit this paragraph.

Reply:
https://github.com/SumoLogic/sumologic-documentation/issues/3294